src/main/flight/mixer_tricopter.c

```diff
@@ -571,7 +571,7 @@ static void tailTuneModeThrustTorque(struct thrustTorque_t *pTT, const bool isTh
     case TT_WAIT_FOR_DISARM:
         if (!ARMING_FLAG(ARMED))
         {
-            float averageServoAngle = pTT->servoAvgAngle.sum / 10.0f / pTT->servoAvgAngle.numOf;
+            float averageServoAngle = pTT->servoAvgAngle.sum / 10.0f / pTT->servoAvgAngle.numOf - 90.0f;
             if (averageServoAngle > 0.5f && averageServoAngle < 40.f) { // TODO: Too wide? or totally unnecessary?
```

Comment:
Here is a little analysis:
A prop with a higher factor must have less drag and must produce more thrust per watt. That is not easy to do.
The prop selected by David W is pretty good. Let's say we can find another one which is double good (but then I think David would have chosen that one already).
If our prop have a factor of 14, I'd say its unlikely we find another one with a factor >30, because I think that would mean that the drag is half.
That corresponds to an angle = 1.9 deg.
Remove the misalignment from bad bench tuning. 1 deg is cleary visible. Gives us 0.9.
So 0.5 (factor=114) should be on the safe side. The current function stops at 0.15 deg but I don't see how that can happen. A prop cannot be that good.Or show me where to buy ;)
At lower angles the factor grow very rapidly into indefinit at 0.

In the other direction; how bad prop can we have? Four times worse? Who would buy it?
A factor at 14/4 corresponds to 16 deg. Add one for misalignment gives 17 deg.
Say 30 to be safe.

So 0.5 .. 30 deg is a reasonable and not unrealistic range.
If we put 0.01 .. 90 there would never be a failed tuning and we can remove the check completely.
Or what did I miss here?